include/crocoddyl/multibody/contacts/contact-2d.hxx

```diff
@@ -70,11 +70,16 @@ void ContactModel2DTpl<Scalar>::calcDiff(const boost::shared_ptr<ContactDataAbst
   d->da0_dx.leftCols(nv).row(1).noalias() -= d->vv_skew.row(2) * d->fXjdv_dq.template bottomRows<3>();
 
   d->da0_dx.rightCols(nv).row(0) = d->fXjda_dv.row(0);
-  d->da0_dx.rightCols(nv).row(0).noalias() += d->vw_skew.row(0) * d->fJf.template topRows<3>();
+  typename MathBase::MatrixXs vw_skew2D(1,2);
```

Comment:
Instead, of this line, add the following line in `include/crocoddyl/core/mathbase.hpp`
```
typedef Eigen::Matrix<Scalar, 1, 2> RowVector2s;
```

and then replace this line with 
```
typename MathBase::RowVector2s vw_skew2D;
```